### PR TITLE
[GORA-638] Add gora-ignite into the gora-tutorial module

### PR DESCRIPTION
**Pull request by joseph6x**
Add support for Ignite in the gora-tutorial module.

As part of my experiments with Gora, I tried to add ignite in the tutorial module using other other databases as example. Please let me know if I missed something.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1012,6 +1012,17 @@
         <version>${project.version}</version>
         <type>test-jar</type>
       </dependency>
+      <dependency>
```
Comment (djkevincr) [inline pom.xml]:
```
 <dependency>
        <groupId>org.apache.gora</groupId>
        <artifactId>gora-redis</artifactId>
        <version>${project.version}</version>
      </dependency>
      <dependency>
        <groupId>org.apache.gora</groupId>
        <artifactId>gora-redis</artifactId>
        <version>${project.version}</version>
        <type>test-jar</type>
      </dependency>
```
Do we need these redis dependencies as part of this PR?

Comment (joseph6x) [reply pom.xml]: Sorry for that. I unintentionally mixed code from other experiments.

**Comment by djkevincr** [comment]
@joseph6x Thank you for your contribution. :) This looks good to me. I will locally test and merge the PR.

**Comment by djkevincr** [comment]
@joseph6x I merged your PR. Thank you for this and looking forward for your future contributions. :)

**Pull request merged**